api/graphql/spaces/space_filtersets.py

```diff
@@ -0,0 +1,25 @@
+import django_filters
+from django.db.models import Q
+
+from permissions.helpers import can_manage_spaces, get_units_with_permission
+
+
+class SpaceFilterSet(django_filters.FilterSet):
+    only_with_permission = django_filters.BooleanFilter(
+        method="get_only_with_permission"
+    )
+
+    def get_only_with_permission(self, qs, property, value):
+        """Returns spaces where the user has any kind of permissions"""
```

Comment:
`Returns spaces where the user has space management permissions.`

Reply:
Fixed! 👍